test.mjs

```diff
@@ -251,6 +251,20 @@ if (test('Retry works')) {
   assert(Date.now() >= now + 50 * (5 - 1))
 }
 
+if (test('spawn timeout')) {
+  let signal = 0
+  $.spawn.timeout = 100
+  $.spawn.killSignal = 'SIGKILL'
```

Comment:
Also, I'm not really like this API. timeout & killSignal seems not related here, but they are.

Reply:
https://nodejs.org/api/child_process.html#child_processspawncommand-args-options

Reply:
I mean the difference between:
```
$.spawn.timeout = 100
$.spawn.killSignal = 'SIGKILL'
```
and 
```
{ timeout: 100
  killSignal: 'SIGKILL'}
```